ui/css/treeherder-custom-styles.css

```diff
@@ -135,6 +135,7 @@
   height: 16px;
   vertical-align: text-top;
   opacity: 0.7;
+  user-select: none;
```

Comment:
I'm not a CSS expert, but my reading of `user-select` says that it only controls selection of text within an element, not alt text. Am I missing something here?

Reply:
When an image gets selected and pasted in a context which doesn't support images, the `alt` text gets used. No image selection == no alt text pasted in editors, Bugzilla, Treeherder's bug filer etc.